src/include/daos_mgmt.h

```diff
@@ -53,6 +53,8 @@ typedef struct {
 	 * daos_pool_info_t		 mgpi_info;
 	 */
 	uuid_t				 mgpi_uuid;
+	/** Pool label */
+	d_string_t                       mgpi_label;
```

Comment:
I was going to say that this is an API breaking change. but it sounds like we only used this in dmg functions, so probably no user was using this before.